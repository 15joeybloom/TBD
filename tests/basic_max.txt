select max(majorid) from student
exit
