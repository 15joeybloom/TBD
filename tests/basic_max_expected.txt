
SQL> 
select max(majorid) from student
max(majorid)
------
    30

SQL> 
exit
